packages/schematics/src/collection/library/schema.json

```diff
@@ -31,6 +31,10 @@
             "value": "none",
             "label": "Typescript [ https://www.typescriptlang.org/ ]"
           },
+          {
+            "value": "react",
+            "label": "React      [ https://angular.io/             ]"
```

Comment:
@FrozenPandaz sorry to butt in, but this PR caught my eye  👁👁. Just wanted to point out changing this URL to https://reactjs.org/.

Reply:
Thanks :) Fixed.

No need to be sorry, butt in whenever you have something to say 👍